test/integration/profile_security_question_test.go

```diff
@@ -0,0 +1,40 @@
+package integration
+
+import (
+    "context"
+    "fmt"
+    "testing"
+
+    "github.com/stretchr/testify/require"
+    "golang.org/x/exp/slog"
```

Comment:
We should be able to change this import to `log/slog` since it was added to the Go standard library in 1.21 👍